app/javascript/src/components/common-prop-types/tag-prop-types.js

```diff
@@ -0,0 +1,11 @@
+import PropTypes from 'prop-types';
+
+const tagPropType = PropTypes.shape({
+  id: PropTypes.string.isRequired,
+  name: PropTypes.string.isRequired,
+  following: PropTypes.bool.isRequired,
+  bg_color_hex: PropTypes.string.isRequired,
+  text_color_hex: PropTypes.string.isRequired,
+}).isRequired;
+
+export default tagPropType;
```

Comment:
Wasn't this used somewhere in the previous state of the PR? If it's not needed anymore, can we remove it?

Reply:
removed